common/src/main/java/tachyon/Constants.java

```diff
@@ -363,6 +363,9 @@
   public static final int DEFAULT_TFS_PERMISSIONS_UMASK = 0022;
   public static final short DEFAULT_TFS_FULL_PERMISSION = (short) 0777;
   public static final short FILE_DIR_PERMISSION_DIFF = (short) 0111;
+  // Group Mapping
+  public static final String SECURITY_GROUP_MAPPING = "tachyon.security.group.mapping";
```

Comment:
one is called `tachyon.security.group.mapping` and the other is `tachyon.security.groups.cache.secs`,
how about let's keep them consistent, either both using `group` or `groups`.